docs/gettsim_objects/means_testing.md

```diff
@@ -4,14 +4,19 @@
 
 The table below gives an overview of all tangible and intangible assets which are
 considered when performing means for several transfers. A cross indicates that the asset
-class is not considered and, hence, deducted from the overall assets of a household.
+class is not considered and, hence, deducted from the overall assets.
 
 This documentation shall help to understand the composition of the
 
 {ref}`basic input variable <input_variables>`
 
-'vermögen_bedürft_bg'. Despite small differences over the transfers, we decided, for
-now, to require only one wealth variable as input and use it for all transfers.
+'vermögen_bedürft'. Despite small differences over the transfers, we decided, for now,
+to require only one wealth variable as input and use it for all transfers.
+
+Note that GETTSIM requests wealth on the individual level as input. This variable is
+then aggregated on the respective grouping level for each transfer. In most data sets
+wealth is only measured on the household level and, hence, needs to be split up on the
+individual level before feeding it into GETTSIM.
```

Comment:
Great. Please add that this is due to the fact that transfers are calculated at the sub-household level (Bedarfsgemeinschaften), which is endogenous. And that if in doubt, robustness checks regarding different distributions of wealth across household members might be in order.